templates/ntp.conf.epp

```diff
@@ -0,0 +1,113 @@
+# ntp.conf: Managed by puppet.
+#
+# Enable next tinker options:
+# panic - keep ntpd from panicking in the event of a large clock skew
+# when a VM guest is suspended and resumed;
+# stepout - allow ntpd change offset faster
+<% if $ntp::_tinker and ($ntp::_panic or $ntp::stepout) {-%>
+tinker<% if $ntp::_panic { %> panic <%= $ntp::_panic %><% } %><%if $ntp::stepout { %> stepout <%=$ntp::stepout %><% } %>
+<% } -%>
+
+<% if $ntp::disable_monitor {-%>
```

Comment:
Would `{ -%>` (with a space) be better? I'm totally up for fixing style in the last minute. I need to do a pass at outstanding PRs and rebase this AGAIN anyways :weary: .